apps/sap-commerce-cloud/src/api/fetchBaseSites.ts

```diff
@@ -1,12 +1,27 @@
 import { baseSiteTransformer } from './dataTransformers';
-import { FieldExtensionSDK } from '@contentful/app-sdk';
+import { config } from '../config';
 
-export async function fetchBaseSites(config: any): Promise<string[]> {
-  const response = await fetch(config.installation.apiEndpoint + '/occ/v2/basesites');
-  if (response.ok) {
-    const responseJson = await response.json();
-    const baseSites = responseJson['baseSites'].map(baseSiteTransformer());
-    return baseSites;
+export async function fetchBaseSites(
+  parameters: any,
```

Comment:
are we concerned at all with implicit/explicit `any` types at all? Wondering if we should define these explicitly? Just for my own curiosity nothing blocking by any means 🙏

Reply:
Personally I'm not a fan of `any`, and would rather we have explicit types/interfaces defined. A lot of `any`'s in this app are leftovers from the external team that originally built this, so seems like more of a shortcut to less TS errors. I can create a type for these parameters while I'm in that code (better to do it now than later imo). 

Good callout 👍

Reply:
@matthew-gordon updated.